pagerduty/resource_pagerduty_service_integration.go

```diff
@@ -111,13 +116,20 @@ func buildServiceIntegrationStruct(d *schema.ResourceData) *pagerduty.Integratio
 		}
 	}
 
-	return serviceIntegration
+	if serviceIntegration.IntegrationEmail == "" {
```

Comment:
@Yannic this is not necessarily an email integration? Just like in customize diff you should also check for `t == "generic_email_inbound_integration"`. I'm actually surprised the tests did not fail, perhaps we should modify the existing tests to confirm this will not fail if service integration is of another type?

Reply:
done